Automattic/jetpack
# Components: Update clicking a tooltip to toggle visibility instead of always show

Fixes #35294

Please note, that this PR changes the default behavior throughout all products that use the component. I'm not sure if the way it was previously was better, but I think this is an improvement.

## Proposed changes:

* Update behavior to close the tooltip when clicking the tooltip icon again.

	<details><summary>Before</summary>
	
	![CleanShot 2024-01-29 at 19 01 05](https://github.com/Automattic/jetpack/assets/11799079/a9b12fe3-21e7-4708-8068-a72faf762715)
	
	
	</details>
	
	<details><summary>Now</summary>
	
	![CleanShot 2024-01-29 at 19 00 47](https://github.com/Automattic/jetpack/assets/11799079/093f4568-ca25-4372-8403-597fb197969d)
	
	
	</details>

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost;
* Check any tooltips (except the Critical CSS one as that's using an older component) and make sure they behave as displayed above (from the **Now** example).
